File: Python/foundation/FormParameter.py

class FormParameter:

	def __init__(self):
		self.key=None
		self.value=None
	
	def __init__(self, key, value):
		self.key=key
		self.value=value
	
	def getKey(self): 
		return self.key
		
	def getValue(self):
		return self.value
	
	def setKey(self,key):
		self.key=key
		
	def setValue(self,value):
		self.value=value
	

File: Ruby/foundation/FormParameter.rb

class FormParameter
  
	def initialize(key=nil, value=nil)
		@key=key
		@value=value
	end
	
	def getKey 
		@key
	end
		
	def getValue
		@value
	end
	
	def setKey(key)
		@key=key
	end
		
	def setValue(value)
		@value=value
	end
	
end
	